.env-sample

```diff
@@ -0,0 +1,3 @@
+export REMOTE_RETRO_GOOGLE_OAUTH_CLIENT_ID="<Client Id>"
```

Comment:
can we kill this? or include it in a separate PR with an explanation of its use? I'm not 100% against it; it just feels unrelated

Reply:
r, if you want to make a case for inclusion in this PR, i'm down to hear it!

Reply:
Oh not a biggie, it can go elsewhere.